bpf/lib/maps.h

```diff
@@ -85,28 +85,6 @@ struct bpf_elf_map __section_maps CONFIG_MAP = {
 };
 #endif
 
-#ifdef ENABLE_IPV4
-struct bpf_elf_map __section_maps PROXY4_MAP = {
```

Comment:
(not sure on this, which is why this is a question - maybe @joestringer can answer as well) - will these maps get cleaned up automatically now that they are removed from the Cilium codebase, or will they continue to reside on the BPF filesystem?

Reply:
Good point, we should add a unit test to `pkg/datapath/maps` to ensure it's cleaned up.

Reply:
I have not addressed this in this PR.